.circleci/config.yml

```diff
@@ -45,83 +40,22 @@ jobs:
           command: |
             git config --global user.email "ci@infinite.red"
             git config --global user.name "Infinite Red"
-      - run:
-          name: Install PNPM for tests
-          command: |
-            sudo chmod 777 /usr/local/bin
-            curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm@7
-            pnpm config set store-dir ../.pnpm-store
       - restore_cache:
           name: Restore ignite dependency cache
           key: ignite-deps-cache-{{ .Environment.IGNITE_DEPS_PACKAGER }}-{{ checksum "boilerplate/package.json" }}-{{ arch }}-{{ .Environment.IGNITE_DEPS_KEY_SUFFIX }}
       - run:
-          name: Run tests
-          command: yarn ci:test
+          name: Run static tests
+          command: yarn format:check & yarn lint & yarn typecheck
+      - run:
+          name: Run jest tests
+          command: yarn test
```

Comment:
Ran this seperately than the other tests because it is slower, and running multiple yarn scripts with `&` will combine all the std outputs to the same line, and it looks weird with jest test runner